### PR TITLE
Adjust semantic HTML equivalent for `rowheader`

### PR DESCRIPTION
**Pull request by jan-san**
#### Summary
Change the semantic HTML equivalent for `rowheader` from `<tr col="row">` to `<th scope="row">`.

#### Motivation
The suggested HTML equivalent is consistent with this section from the [rowheader role page](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/rowheader_role):

> It is a structural equivalent to the `<th>` element with a scope of row, `<th scope="row">`.

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

**Comment by jan-san** [comment]
@wbamberg I restructured the section as you suggested.

File: files/en-us/web/accessibility/aria/roles/index.md [hunk 1]
```diff
@@ -35,7 +35,29 @@ Document Structure roles are used to provide a structural description for a sect
...
+- [img](/en-US/docs/Web/Accessibility/ARIA/Roles/img_role) (use {{HTMLElement('img')}} or {{HTMLElement('picture')}} instead)
+- [list](/en-US/docs/Web/Accessibility/ARIA/Roles/list_role) (use either {{HTMLElement('ul')}} or {{HTMLElement('ol')}} instead)
+- [listitem](/en-US/docs/Web/Accessibility/ARIA/Roles/listitem_role) (use {{HTMLElement('li')}} instead)
+- [meter](/en-US/docs/Web/Accessibility/ARIA/Roles/meter_role)  (use {{HTMLElement('meter')}} instead)
+- [row](/en-US/docs/Web/Accessibility/ARIA/Roles/row_role) (use the {{HTMLElement('tr')}} with {{HTMLElement('table')}})
+- [rowgroup](/en-US/docs/Web/Accessibility/ARIA/Roles/rowgroup_role) (use {{HTMLElement('thead')}}, {{HTMLElement('tfoot')}} and {{HTMLElement('tbody')}})
+- [rowheader](/en-US/docs/Web/Accessibility/ARIA/Roles/rowheader_role) (use `{{HTMLElement('th','&lt;th scope="row">')}}`)
+- [separator](/en-US/docs/Web/Accessibility/ARIA/Roles/separator_role) (use {{HTMLElement('hr')}} if it doesn't have focus)
+- [table](/en-US/docs/Web/Accessibility/ARIA/Roles/table_role) (use {{HTMLElement('table')}})
+- [term](/en-US/docs/Web/Accessibility/ARIA/Roles/term_role) (use {{HTMLElement('dfn')}}
```
Comment (wbamberg) [inline files/en-us/web/accessibility/aria/roles/index.md:58]:
```suggestion
- [term](/en-US/docs/Web/Accessibility/ARIA/Roles/term_role) (use {{HTMLElement('dfn')}})
```

Comment (jan-san) [reply files/en-us/web/accessibility/aria/roles/index.md]: oh, good catch :+1:

**Review by wbamberg — APPROVED** [review]
👍 thank you @jan-san , this is so much more readable!

**Pull request merged**